src/index.js

```diff
@@ -1,3 +1,3 @@
-import Slider from './slider';
+import Slider from './Slider';
```

Comment:
this should be fixed in a new release I put out. Please let me know if it is not.